data/autoyast_qam/12-common_base_installation.xml.ep

```diff
@@ -238,6 +238,9 @@
     <global>
       <timeout config:type="integer">-1</timeout>
     </global>
+    % if ($check_var->('UEFI', '1') and $check_var->('ARCH', 'x86_64')) {
+    <loader_type>grub2-efi</loader_type>
+    % }
   </bootloader>
```

Comment:
In this case,  we may miss `aarch64`, can you please run VR on aarch64 to see if any regression issue?

Reply:
Verification run on aarch64: https://openqa.suse.de/tests/13545686